resources/asciidoctor/lib/docbook_compat/convert_document.rb

```diff
@@ -17,21 +22,6 @@ def convert_document(doc)
       html + "\n"
     end
 
-    ##
```

Comment:
I moved this one to "make room" for the new features in this file. rubocop  doesn't like when files get too big. Neither do I, but it is better at noticing.